azure/README.md

```diff
@@ -69,6 +69,38 @@ docker build -t azure \
 If you want to use a different base image, then you'll need to provide additional build-args:
 `PUBLISHER`, `OFFER`, `SKU`
 
+Sometimes using the marketplace image requires accepting a licensing agreement and also using a published plan.
+Following [link](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage) provides more detail.
+
+For example using the CentOS 8.5 image from eurolinux publisher requires a plan and license agreement.
+You'll need to first get the URN:
+```
+az vm image list --location $AZURE_REGION --publisher eurolinuxspzoo1620639373013  --offer centos-8-5-free --sku centos-8-5-free --all --output table
+```
+Then you'll need to accept the agreement:
+```
+az vm image terms accept --urn eurolinuxspzoo1620639373013:centos-8-5-free:centos-8-5-free:8.5.5
+```
```

Comment:
can it be done from within the dockerfile?

Reply:
Technically it can be done, but it's better to keep it independent of the Dockerfile for the following reasons:
1. Requires az cli to be available in the container image
2. Requires authentication to be performed inside the container
3. Not every image requires license acceptance
4. The acceptance is global for the subscription. So preferred base images should already be available and attached to the subscription.